app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      include Wor::Paginate
+      def index
+        rents = Rent.where(['user_id = :uid', { uid: params[:user_id] }])
+        render_paginated rents, each_serializer: RentIndexSerializer
+      end
+
+      def create
+        if current_user.id != params[:rent][:user_id]
+          render json: { message: 'User id not allowed' }, status:	:unprocessable_entity
+          return
+        end
+        rent = Rent.new(rent_params)
+        if rent.save
```

Comment:
We can try something `if modifier` or `unless modifier` here, is really useful and clear.
Instead of asking for `rent.save`, we can do something like: 
`return render json: rent.errors, status: :unprocessable_entity unless rent.save`
And then below that do
`render json: rent, serializer: RentIndexSerializer, status: :created`

Reply:
Check this out here https://www.tutorialspoint.com/ruby/ruby_if_else.htm